src/components/ProductCard/ProductCard.scss

```diff
@@ -16,7 +16,7 @@
 
   &__img-container {
     display: flex;
-    justify-self: center;
+    margin: 0 auto;
```

Comment:
it seems like it doesn't work :cry:

Reply:
Oh... I`ve just seen it.. But I figured it out but my own)) justify-self seems the most suspicious to me))